BookingQueueSubscriber/BookingQueueSubscriber.Services/MessageHandlers/NewParticipantMultidayHearingConfirmationHandler.cs

```diff
@@ -45,6 +48,7 @@ public async Task HandleAsync(NewParticipantMultidayHearingConfirmationEvent eve
             };
 
             await _bookingsApiClient.UpdatePersonUsernameAsync(message.ContactEmail, message.Username);
+            await _videoApiService.UpdateParticipantUsernameWithPolling(message.HearingId, newUser.UserName, message.ContactEmail);
```

Comment:
good find! Could we add an assertion for the username being updated?

if we add this:

`_videoApiService.UpdateParticipantDetailsCount.Should().Be(1);`

to

`Should_send_multiday_confirmation_notifications_for_new_participant`

that should cover it I think